node/flatpak_node_generator/providers/yarn.py

```diff
@@ -86,13 +86,17 @@ def parse_package_section(self, lockfile: Path, section: List[str]) -> Package:
                 values = self.unquote(values_str).split(' ')
                 integrity = Integrity.parse(values[0])
 
-        assert version and resolved, line
+        assert version, section
 
         source: PackageSource
-        if self.is_git_version(resolved):
-            source = self.parse_git_source(version=resolved)
+        if name_version.startswith('link:'):
```

Comment:
Yarn can use `file:` too, `link:` is a separate thing.

Reply:
Used a regex for the version constraint prefix check/removal.